openpathsampling/engines/dynamics_engine.py

```diff
@@ -12,13 +12,19 @@
 
 from openpathsampling.netcdfplus import StorableNamedObject
 
-from snapshot import BaseSnapshot
-from trajectory import Trajectory
+from .snapshot import BaseSnapshot
+from .trajectory import Trajectory
 
-from delayedinterrupt import DelayedInterrupt
+from .delayedinterrupt import DelayedInterrupt
 
 logger = logging.getLogger(__name__)
 
+if sys.version_info > (3, ):
+    property_name = lambda p: str(p.fget.__name__)
+else:
+    # property_name = lambda p: str(p.fget.func_name)
+    property_name = lambda p: str(p.fget.__name__)
+
```

Comment:
Ah -- since this is also available in 2.7, we should just use `fget.__name__` instead of wrapping a function here. See also line 272. I had kept the use of `fget.func_name` that had been in the original code, guess it is already aliased.